controllers/storage/minio.go

```diff
@@ -83,20 +119,28 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 	if isReady {
 		err := createDefaultBucket()
 		if err != nil {
-			return minioNotReadyStatus(CreateDefaultBucketeError, err.Error()), err
+			return minioNotReadyStatus(CreateDefaultBucketError, err.Error()), err
 		}
-		return m.ProvisionInClusterSecretAsOss(&minioCR)
+		return m.ProvisionInClusterSecretAsS3(&minioCR)
 	}
 
-	return nil, nil
+	return minioUnknownStatus(), nil
 }
 
 func createDefaultBucket() error {
 	panic("implement me")
```

Comment:
Why not implemented it?

Reply:
sorry,  I forgot.  I will create another PR to implement it.